scripts/tapir/deploy_root.py

```diff
@@ -25,16 +21,13 @@ def main():
     eth-ape                   0.6.20
     """
 
-    deployer = Deployer.from_yaml(
-        filepath=CONSTRUCTOR_PARAMS_FILEPATH,
-        verify=VERIFY
-    )
+    deployer = Deployer.from_yaml(filepath=CONSTRUCTOR_PARAMS_FILEPATH, verify=VERIFY)
 
     reward_token = deployer.deploy(project.TapirStakingToken)
 
     mock_threshold_staking = deployer.deploy(project.TestnetThresholdStaking)
 
-    proxy_admin = deployer.deploy(OZ_DEPENDENCY.ProxyAdmin)
+    proxy_admin = deployer.deploy(OZ_DEPENDENCY.ProxyAdmin, deployer)
```

Comment:
The new version of `TransparentUpgradeableProxy` deploys a default `ProxyAdmin` so this line will need to be removed.  I can address this in #164

Reply:
I suppose we would want to reuse proxy admin between taco child and coordinator, is this possible with the new changes?

Reply:
That does not appear to be possible (or necessary) anymore: https://github.com/OpenZeppelin/openzeppelin-contracts/blob/932fddf69a699a9a80fd2396fd1a2ab91cdda123/contracts/proxy/transparent/TransparentUpgradeableProxy.sol#L78